extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/DimmerSwitch.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hue" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Hue Dimmer Switch -->
+    <thing-type id="0820">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="bridge" />
+        </supported-bridge-type-refs>
+
+        <label>Dimmer Switch</label>
+        <description>A dimmer switch</description>
+
+        <channels>
+            <channel id="dimmer_switch" typeId="dimmer_switch" />
+            <channel id="last_updated" typeId="last_updated" />
+        </channels>
+
+        <representation-property>uniqueId</representation-property>
+
+        <config-description>
+            <parameter name="lightId" type="text">
```

Comment:
the lightId and sensorId are basically the same thing on the bridge, and i didn't want to introduce a new id-parameter which adds nothing. Maybe the lightId should be changed to objectId or something, but i dont know whether this would be backwards compatible

Reply:
I know, but isn't it a little bit confusion to set a property `lightId` on a sensor device?